cub/cub/device/dispatch/tuning/tuning_adjacent_difference.cuh

```diff
@@ -0,0 +1,85 @@
+/******************************************************************************
```

Comment:
Nitpick: I would drop the prefix "tuning_" from the files, they are already in the "tuning" folder

Reply:
I totally agree and would do the same for `device_` and `dispatch_`. However, then we have three files `merge_sort.cuh` with different meanings depending on the directory. I think that's the reason why there are named this way.